infoschema/tables.go

```diff
@@ -1131,14 +1134,25 @@ var tableInspectionResultCols = []columnInfo{
 	{"DETAILS", mysql.TypeVarchar, 256, 0, nil, nil},
 }
 
+var tableMetricTablesCols = []columnInfo{
+	{"METRIC_NAME", mysql.TypeVarchar, 64, 0, nil, nil},
```

Comment:
It's better to name this column as `TABLE_NAME`.

Reply:
great,done.